### ISSUE TITLE
Unexpected keyword argument 'bubble' on master, pipy broken

### ISSUE THREAD

**Issue by satreix** [slash-testing/slash#372]
Hi,

I'm starting off with slash and noticed the pipy version is broken.

```
(venv) satreix@nova ~/work/slash-test » slash run test_addition.py
[...]
File "lib/python3.4/site-packages/slash/log.py", line 172,
in _get_file_log_handler
    handler = logbook.NullHandler(bubble=False)
TypeError: __init__() got an unexpected keyword argument 'bubble'
```

The develop branch solves this issues but master should be usable out of the box right ?

**Comment by vmalloc** [comment]
Ack - working on it. Sorry for the inconvenience.

**Comment by vmalloc** [comment]
Fixed in 1f6334effe66e88e6b054ae7f2be08538e0ac16b, releasing 0.18.2 now
